python/lsst/phosim/utils/phosim_repackager.py

```diff
@@ -158,6 +193,14 @@ def repackage(self, phosim_amp_files, out_dir="."):
                 amp.getBBox(), flipx=amp.getRawFlipX(), flipy=amp.getRawFlipY()
             )
 
+            # Set the filter information.
+            # For lsstCam, the phosim filter names (ugrizy)
+            # are appropriate, but for comCam they need
+            # to have a different name as in obs_lsst filters.py
+            if self.instrument == "comCam":
+                phosimFilter = hdu.header["FILTER"]
+                hdu.header["FILTER"] = translate_comcam_filter(phosimFilter)
+
```

Comment:
I felt this part of code is duplicated in `repackage()` and `repackage_eimage()`. Do you want to consider to abstract this such as `updateComCamSpecificData(header)` and put the related specific setting/data of ComCam into this function to reduce the duplication of code (this also means you could remove the `translate_comcam_filter()` and put the logic into the `updateComCamSpecificData(header)`. The benefit is that you will only need to modify one function if you find there would be other data needs to change in additional to filter in ComCam.

Reply:
Moved from `translate_comcam_filter(phosimFilter)` to  `updateComCamSpecificData(header)` function  that takes the FITS header and does appropriate updates (currently updating the FILTER header). Added appropriate test.